LoopTests/Models/RemoteCommandTests.swift

```diff
@@ -37,7 +37,7 @@ class RemoteCommandTests: XCTestCase {
         ]
```

Comment:
Added new tests for the new cases. I also needed to adjust a few unrelated due to the new `defaultAbsorptionTime` parameter.